blake2/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["cryptography", "no-std"]
 
 [dependencies]
 digest = { version = "0.10.7", features = ["mac"] }
+zeroize_crate = { package = "zeroize", version = "1", default-features = false, optional = true }
```

Comment:
I think we can simply use `zeroize = { version = "1", default-features = false, optional = true }`?

Reply:
I'm assuming you mean by removing the zeroize feature, which does seem to not be needed in this use-case. Sorry, I'm unfamiliar with writing tomls with such old support :sweat_smile: